app/services/utilities/employment_income_monthly_equivalent_calculator.rb

```diff
@@ -49,22 +49,26 @@ def weekly_to_monthly(value)
       (value * 52 / 12).round(2)
     end
 
-    def blunt_average(_value)
-      @employment.employment_payments.sum(&:gross_income) / 3
+    def blunt_average(value)
+      (@employment.employment_payments.sum(&value) / @employment.employment_payments.count).round(2)
     end
 
     def update_payments(calc_method)
       @employment.employment_payments.each do |payment|
         payment.update(
-          gross_income_monthly_equiv: __send__(calc_method, payment.gross_income),
-          tax_monthly_equiv: __send__(calc_method, payment.tax),
-          national_insurance_monthly_equiv: __send__(calc_method, payment.national_insurance),
+          gross_income_monthly_equiv: __send__(calc_method, calculation_value(calc_method, payment, "gross_income")),
```

Comment:
Should be able to use a symbol here?!

```suggestion
          gross_income_monthly_equiv: __send__(calc_method, calculation_value(calc_method, payment, :gross_income)),
```

and similarly for others below.